cypress/steps/mapping-steps.ts

```
@@ -287,7 +290,7 @@ class MappingSteps {
 
   // Notifications
   static getNotification() {
-    return cy.get('.mat-simple-snackbar');
+    return cy.get('.mat-simple-snackbar', {timeout: 10000});
```

I don't like this. Unconditional wait usually means you don't know what happens with the app. Can't we have a proper check here?

---

The app is waiting for the toaster message to disappear. When under load it takes more than 4 sec (default timeout) to hide it. This timeout is not a wait. This is the maximum time cypress will wait for the toaster.